requirements.txt

```diff
@@ -16,3 +16,4 @@ setuptools==30.4.0
 six==1.10.0
 tabulate==0.7.5
 vcrpy==1.10.3
+-e git+https://github.com/Azure/azure-sdk-for-python.git@azurestack#egg=azure-mgmt-storage&subdirectory=azure-mgmt-storage
```

Comment:
This will be removed before merging to master but needed currently as SDK not released.